src/OrbitGl/GraphTrack.cpp

```diff
@@ -281,7 +284,7 @@ void GraphTrack<Dimension>::DrawSeries(PrimitiveAssembler& primitive_assembler,
   // We skip the last element because we can't calculate time passed between last element
   // and the next one.
   while (current_it != last_it) {
-    std::array<double, Dimension> cumulative_values{current_it->second};
+    std::vector<double> cumulative_values = current_it->second;
```

Comment:
Same here, can you add a check that verifies the size of the vector is the same as the dimension?

Reply:
I will add the check outside the while loop as all the value vectors in `entries` have the same dimension: this is guaranteed by the check in `series_.AddValue`.

Thanks for the suggestion!